src/NUnitFramework/nunitlite/CommandLineOptions.cs

```diff
@@ -169,6 +169,7 @@ private static IEnumerable<string> GetArgsFromFile(string filename)
         public string InputFile { get; private set; }
 
         public IList<string> TestList { get; } = new List<string>();
+        public IList<string> PreFilters { get; } = new List<string>();
```

Comment:
The whole list is the prefilter, so singular?

Reply:
In this code, it's just a list of names, similar to `TestList`. Later all the entries get added to a `PreFilter`  the same way that the `TestList` items get added to a `TestFilter`. So maybe we should call this a `FilterList`?